pkgs/development/python-modules/confuse/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, isPy3k
+, pyyaml
+}:
+
+buildPythonPackage rec {
+  pname = "confuse";
+  version = "1.0.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "5977c6a876b179f8c86ecaf530cc24bf00baaad028c3350b502ee6949536188c";
+  };
+
+  propagatedBuildInputs = [ pyyaml ];
+
+  doCheck = false;
```

Comment:
why?

Reply:
I tried it with `checkInputs = [ tox ];` as this package requires `tox` for testing multiple interpreters (see https://github.com/beetbox/confuse/blob/1619d3ac33d2959cd5f4aa9ad725935b790ad12f/setup.py#L84 and https://github.com/beetbox/confuse/blob/1619d3ac33d2959cd5f4aa9ad725935b790ad12f/tox.ini#L7). However, it fails the check phase with `Error creating virtualenv.`. 
I looked at other projects, which rely on `tox`, but failed to get it to work.

Reply:
tox is not what you want, instead you want the other test inputs like pytest or what ever they use, then run the test suite with those. Generally people add tox as a convience to their own workflows, but it's relevant in nixpkgs since a python interpreter is parameterized to begin with.

Reply:
thank you @jonringer for your valuable input! After fiddling with getting the tests running I found that this perticular version (`1.0.0`) does not yet include `test/__init__.py` in `MANIFEST.in`. According to their corresponding issue (https://github.com/beetbox/confuse/issues/54), they want to deliver this in the new version `1.0.1`, which is still in preparation.

Reply:
it's common for some people to exclude tests when packaging, but if they have a github repo, then you can just pull source from the repo